README.md

```diff
@@ -372,25 +365,25 @@ vendor/bin/ecs check --config vendor/rector/rector/ecs-after-rector.neon
 vendor/bin/ecs check --config vendor/rector/rector/ecs-after-rector.neon --fix
 ```
 
-### More Detailed Documentation
+## More Detailed Documentation
 
 - [How to Create Own Rector](/docs/HowToCreateOwnRector.md)
 - [Service Name to Type Provider](/docs/ServiceNameToTypeProvider.md)
 
 
-### How to Contribute
+## How to Contribute
 
 Just follow 3 rules:
 
 - **1 feature per pull-request**
 - **New feature needs tests**
-- Tests, coding standard and PHPStan **checks must pass**
+- Tests, Coding Standards and PHPStan **checks must pass**:
 
     ```bash
     composer all
     ```
 
-    Don you need to fix coding standards? Run:
+    Don you need to fix Coding Standards? Run:
```

Comment:
This is common name, so small caps are correct.
In case of packages name it would be uppercase as you write